sbe-tool/src/main/java/uk/co/real_logic/sbe/generation/TargetCodeGeneratorLoader.java

```diff
@@ -52,7 +52,8 @@ public CodeGenerator newInstance(final Ir ir, final String outputDir)
                 "true".equals(System.getProperty(JAVA_GROUP_ORDER_ANNOTATION)),
                 "true".equals(System.getProperty(JAVA_GENERATE_INTERFACES)),
                 "true".equals(System.getProperty(DECODE_UNKNOWN_ENUM_VALUES)),
-                new PackageOutputManager(outputDir, ir.applicableNamespace()));
+                "true".equals(System.getProperty(TYPE_PACKAGE_OVERRIDE)),
```

Comment:
Should this default to `true`?

Reply:
unless `TYPE_PACKAGE_OVERRIDE` is explicitly set to `true`, the generator will not respect package overrides, therefore behave as before this change - unless I am missing something here?